pom.xml

```diff
@@ -1659,6 +1667,49 @@ import org.apache.commons.codec.digest.*;
 
     </profile>
 
+    <!-- Generates coverage information -->
+    <profile>
+
+      <id>coverage</id>
+
+      <build>
+        <plugins>
+
+          <plugin>
+            <groupId>org.jacoco</groupId>
+            <artifactId>jacoco-maven-plugin</artifactId>
+            <configuration>
+              <includes>
+                <include>org/apache/flume/**</include>
```

Comment:
I first want to see Flume merging `logging-parent`, before seeing the other way around.
```suggestion
```